scripts/postgres_test_setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# First, start up the container
+docker run --name aqueduct-postgres -e POSTGRES_PASSWORD=aqueduct -d postgres -p 5432:5432
+# TODO: Figure out how to do commands below in bash script.
+# Open terminal inside the container
+docker exec -it <container_id> bash
```

Comment:
I'm not sure how to run these commands through a bash script.
Need to google around a bit, my unix fu is failing me right now :)

Reply:
ChatGPT has me covered here:
<img width="890" alt="image" src="https://user-images.githubusercontent.com/1031759/216800966-82cd7510-695f-4382-bb99-633fa8beb984.png">